pgql-lang/src/test/java/oracle/pgql/lang/PrettyPrintingTest.java

```diff
@@ -667,6 +667,16 @@ public void testLateralSubquery2() throws Exception {
     checkRoundTrip(query);
   }
 
+  @Test
+  public void testWalkTrailAcyclicSimple() throws Exception {
+    String query = "SELECT COUNT(*) " //
```

Comment:
how is e.g `MATCH ANY (n) -[e2]->* (m)` printed, i.e is the implicit `WALK` printed?
Would it make sense to add it here?

Reply:
I think it's clearer without the WALK keyword since not everyone may know what a WALK is (so it's not printing it a.t.m.).
Same with ONE ROW PER MATCH.. leaving it away to keep things simple.